### PR TITLE
[SPARK-9052][SparkR] Fix comments after curly braces

### PR DESCRIPTION
**Pull request by yu-iskw**
[[SPARK-9052] Fix comments after curly braces - ASF JIRA](https://issues.apache.org/jira/browse/SPARK-9052)

This is the full result of lintr at the rivision:011551620faa87107a787530f074af3d9be7e695.
[[SPARK-9052] the result of lint-r at the revision:011551620faa87107a787530f074af3d9be7e695](https://gist.github.com/yu-iskw/e7246041b173a3f29482)

This is the difference of the result between before and after.
https://gist.github.com/yu-iskw/e7246041b173a3f29482/revisions

### CODE REVIEW

**Comment by yu-iskw** [comment]
@shivaram Could you review it? Thanks!

File: R/pkg/R/schema.R [hunk 1]
```diff
@@ -69,10 +69,11 @@ structType.structField <- function(x, ...) {
...
   cat("StructType\n",
-      sapply(x$fields(), function(field) { paste("|-", "name = \"", field$name(),
-                                           "\", type = \"", field$dataType.toString(),
-                                           "\", nullable = ", field$nullable(), "\n",
-                                           sep = "") })
+      sapply(x$fields(), function(field) {
+             paste("|-", "name = \"", field$name(),
+                   "\", type = \"", field$dataType.toString(),
+                   "\", nullable = ", field$nullable(), "\n",
+                   sep = "") })
```
Comment (sarutak) [inline R/pkg/R/schema.R]: It's not about style for comments but how about like as follows?
```
      sapply(x$fields(),
                 function(field) {
                   paste("|-", "name = \"", field$name(), 
                             "\", type = \"", field$dataType.toString(),
                             "\", nullable = ", field$nullable(), "\n",
                             sep = "")
                 })
```

@shivaram Do you have any suggestion?

File: R/pkg/R/utils.R [hunk 2]
```diff
@@ -440,19 +446,23 @@ processClosure <- function(node, oldEnv, defVars, checkedFuncs, newEnv) {
         # Namespaces other than "SparkR" will not be searched.
         if (!isNamespace(func.env) ||
               (getNamespaceName(func.env) == "SparkR" &&
-              !(nodeChar %in% getNamespaceExports("SparkR")))) {  # Only include SparkR internals.
+              !(nodeChar %in% getNamespaceExports("SparkR")))) {
```
Comment (sarutak) [inline R/pkg/R/utils.R]: According to the style guide (http://google-styleguide.googlecode.com/svn/trunk/Rguide.xml#indentation),  we should align this line with `!isNamespace` right?

Maybe, we can see the same issue in various places and should open another PR about it.

File: R/pkg/R/schema.R (hunk 1 above)
Comment (sarutak) [inline R/pkg/R/schema.R]: The result of rendering above is broken (bad indentation)...
I'll show you what I mean later.

Comment (sarutak) [inline R/pkg/R/schema.R]: This is what I mean.
https://github.com/sarutak/spark/blob/SPARK-9052/R/pkg/R/schema.R#L78

Comment (shivaram) [inline R/pkg/R/schema.R]: Yeah I think @sarutak's code looks pretty good. One minor nit in that is that in Line 79, the comma can be moved to the previous line.

File: R/pkg/R/utils.R (hunk 2 above)
Comment (shivaram) [inline R/pkg/R/utils.R]: Hmm the nodeChar check is in the same paren as the getNamespaceName here ? If so the current alignment looks fine to me

Comment (sarutak) [inline R/pkg/R/utils.R]: Ah, right. The `nodechar` check is still in the proper place. How about `(getNamespaceName`? Should it be aligned with `!isNamespace` ?

Comment (shivaram) [inline R/pkg/R/utils.R]: Hmm yes - the `(` in `getNameSpace` could be aligned with the previous line I guess. However when we have nested conditions I actually think its more readable to align the inner condition like a new statement (with 2 spaces) as it is done here.

Comment (sarutak) [inline R/pkg/R/utils.R]: Yeah, nested conditions should be aligned with 2 spaces.

**Comment by yu-iskw** [comment]
@sarutak @shivaram Thank you for your comment. I have modified them.

**Comment by sarutak** [comment]
retest this please.

**Comment by shivaram** [comment]
Thanks for the update @yu-iskw -- Changes look good to me. @sarutak Can you take a final pass and merge ?

**Comment by sarutak** [comment]
LGTM as well. Merging into master. Thanks @yu-iskw !

**Comment by yu-iskw** [comment]
Thank you for merging it?

**Pull request closed without merging**